README.md

```diff
@@ -260,8 +284,8 @@ Indeed, `Wrapper` can now be made an instance of `MonadState`:
 
 ```haskell
 instance Has (State s) sig m => MTL.MonadState s (Wrapper s m) where
-  get = Control.Effect.State.get
-  put = Control.Effect.State.put
+  get = Control.Carrier.State.Strict.get
+  put = Control.Carrier.State.Strict.put
```

Comment:
Step five: we fix any stale references. That’s all I found! Not bad given how much change the library has undergone.